docs/developer/provision-cluster-and-registry.md

```diff
@@ -0,0 +1,75 @@
+# Provision cluster and OCI registry
+
+In this document, you will learn how to set up a cluster in different environments.
+
+For a dual cluster setup, with control plane (KCP) and Kyma runtime (SKR); create **two separate clusters** following the instructions below.
```

Comment:
In General, yes this works - But maybe it could be a bit confusing for some users what things they need to adapt in a two cluster setup (i.e. having different cluster names). I would prefer also to have a small hints at such points. Just to have it bulletproof 👍🏻